client/i18n/locales/english/intro.json

```diff
@@ -2153,7 +2269,12 @@
         ]
       },
       "cfyv": { "title": "275", "intro": [] },
-      "sgau": { "title": "276", "intro": [] },
+      "quiz-javascript-dynamic-programming": {
+        "title": "JavaScript Dynamic Programming Quiz",
```

Comment:
same here.
we could drop javascript and just leave it to dynamic programming

Reply:
✅ Added in next commit.